test/gui/shared/scripts/pageObjects/AccountSetting.py

```diff
@@ -157,3 +163,7 @@ def isLogDialogVisible():
         except:
             pass
         return visible
+
+    @staticmethod
+    def openSettingsTab():
+        squish.clickButton(squish.waitForObject(AccountSetting.SETTINGS_TAB))
```

Comment:
Please see https://github.com/owncloud/client/blob/master/test/gui/shared/scripts/pageObjects/Toolbar.py
We have similar method there and opening settings tab should also go there